src/api.js

```diff
@@ -256,27 +256,27 @@ const moveFile = async ({selectedFile, siteName, resourceName, folderName, subfo
     return await axios.post(apiEndpoint, params)
 }
 
-// Images
-const getImages = async (siteName, customPath) => {
-    const resp = await axios.get(`${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/files/${customPath ? encodeURIComponent(`images/${customPath}`) : 'images'}`);
+// Media
+const getMedia = async (siteName, customPath, mediaType) => {
+    const resp = await axios.get(`${process.env.REACT_APP_BACKEND_URL}/sites/${siteName}/files/${customPath ? encodeURIComponent(`${mediaType}/${customPath}`) : mediaType}`);
     const { directoryContents } = resp.data;
 
-    let respImages = []
+    let respMedia = []
     let respDirectories = []
     directoryContents.forEach((fileOrDir) => {
         const modifiedFileOrDir = { ...fileOrDir, fileName: fileOrDir.name }
```

Comment:
variable name isn't very clear to me, why is this `modifiedFileOrDir` if this is a `GET` call?

Reply:
Renamed in https://github.com/isomerpages/isomercms-frontend/pull/442/commits/641ee75d585935e5503a6409169b336a92669fde